src/inference/tests/unit/cpu_stream_info_test.cpp

```diff
@@ -361,7 +372,31 @@ LinuxCpuStreamTypeCase _1sockets_4cores_binding = {
         MAIN_CORE_PROC,
     },
     {0, 0, 0, 0},
+    {1, 1, 1, 1},
 };
+
+LinuxCpuStreamTypeCase _1sockets_4cores_hyperthreading = {
```

Comment:
I think this test case should correspond to _1sockets_4cores_nobinding. The parameter _core_type should changed to ALL_PROC.

Reply:
thanks, updated